packages/todo_api/lib/src/models/todo.dart

```diff
@@ -1,18 +1,41 @@
 import 'package:equatable/equatable.dart';
+import 'package:uuid/uuid.dart';
 
 class Todo extends Equatable {
   final String? id;
   final String name;
   final String description;
   final bool isConcluded;
 
-  const Todo({
-    this.id,
+  Todo({
+    String? id,
     required this.name,
     this.description = '',
     this.isConcluded = false,
-  });
+  })  : assert(
+          id == null || id.isNotEmpty,
```

Comment:
If id cannot be null, better to just make it required?

Reply:
Make sense! i fixed the code to improve this validation